### PR TITLE
Test mockup PR

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

PR that walks through git workflow with authoring tools

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).

### CODE REVIEW

File: website/docs/docs/environments-in-dbt.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ pagination_next: null
 
 In software engineering, environments are used to enable engineers to develop and test code without impacting the users of their software. Typically, there are two types of environments in dbt:
 
+Test mockup PR.
```
Comment (nghi-ly) [inline website/docs/docs/environments-in-dbt.md:10]: asdfassdf

Comment (nghi-ly) [inline website/docs/docs/environments-in-dbt.md:10]: second time

File: website/docs/docs/environments-in-dbt.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ pagination_next: null
...
+
+Second edit for mockup.
```
Comment (nghi-ly) [inline website/docs/docs/environments-in-dbt.md:12]: second time: another comment

**Review by nghi-ly — COMMENTED** [review]
thanks for reviewing! left a cpl non-blocking suggs and approved. please lemme know if you have any questions about them!

**Review by nataliefiann — APPROVED** [review]
Hiya @nghi-ly 

Thanks for raising this PR. This looks great! I have no suggestions and will approve for you. Please let me know of there's anything else I can do.

Kind Regards

**Comment by nghi-ly** [comment]
thank you so much @nataliefiann for the [mock pr] review!

**Comment by nghi-ly** [comment]
closing as this was a mock walkthrough

**Pull request closed without merging**